Coalition.rte/Devices/Explosives/ClusterGrenade/ClusterGrenade.ini

```diff
@@ -112,13 +112,13 @@ AddDevice = TDExplosive
 	AddToGroup = Bombs - Grenades
 	AddToGroup = Bombs
 	Description = Explosive cluster grenade.  Awesome power!  Blows up spreading many explosive clusters after a 4 second delay.
-	Mass = 5
+	Mass = 3
 	RestThreshold = -500
 	HitsMOs = 1
 	GetsHitByMOs = 1
 	SpriteFile = ContentFile
 		FilePath = Coalition.rte/Devices/Explosives/ClusterGrenade/ClusterGrenade.png
```

Comment:
You deleted this file too.
![uzira-confused](https://user-images.githubusercontent.com/50018358/127309225-aecee7e8-5f50-4113-87e5-b3554464eb51.png) x2